quarkchain/tools/config_p2p.py

```diff
@@ -0,0 +1,38 @@
+"""
+    python config_p2p.py <64 character hex>
```

Comment:
was thinking finding a way to combine `config_slave.py`, `config_p2p.py` and `miner_address.py`

will do in another day